app/views/services/_flow_layout.html.erb

```diff
@@ -22,7 +22,7 @@
   <% else %>
 
     <% if item[:type] == 'flow.branch' %>
-      <article class="flow-item flow-branch" aria-label="TODO-need-editable-label" data-fb-id="<%= item[:uuid] %>">
+      <article class="flow-item flow-branch" aria-label="<%= item[:title] %>" data-fb-id="<%= item[:uuid] %>">
```

Comment:
I'd personally expect aria-label to have better use if it was to describe what this entity is, rather than just duplicating the item title. Also, if we're just using the item title then we might as well use aria-labelledby attribute instead. As per the previous comment on another file, however, it's not a big enough issue to block merging. We can improve things later.

Reply:
Agree its not perfect - I was only looking at this the other day and questioning whether the `<article>` should even have the `aria-label` attribute... but I think I figured *something* was better than 'TODO' and that we can iron out accessibility stuff later.